test/JDBC/expected/latest__verification_cleanup__13_4__sys-databases-dep-vu-verify.out

```diff
@@ -0,0 +1,39 @@
+SELECT name, compatibility_level, collation_name FROM sys_databases_view_dep_vu_prepare
+GO
+~~START~~
+text#!#tinyint#!#nvarchar
+db_sys_databases_dep_vu_prepare#!#<NULL>#!#bbf_unicode_cp1_ci_as
```

Comment:
This is expected because it will be using a deprecate version of the view whereas functions and procedures will use the new version (since they are runtime-based). This deprecated view will not have 120 as a compatibility level value. 

compatibilty _level is null because it's from a dependent view from a previous version. During a previous release we corrected `sys.databases` ' datatypes and added the 120 compatibility level value (can be found here: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/347). We used the method where we renamed the object as a deprecated view. However, this view is still pointing to `sys.databases_deprecated_2_1_0` which does not include the 120 compatibility level value changes.

Reply:
Got it. Thanks